Source/WebCore/html/OffscreenCanvas.cpp

```diff
@@ -462,7 +462,7 @@ void OffscreenCanvas::queueTaskKeepingObjectAlive(TaskSource source, Function<vo
 
 void OffscreenCanvas::dispatchEvent(Event& event)
 {
-    EventDispatcher::dispatchEvent({ this }, event);
+    EventDispatcher::dispatchEvent(std::initializer_list<EventTarget*>({ this }), event);
```

Comment:
I'm curious: Why prefer std::initializer_list to std::array in cases like this?

Reply:
Hmm, They are very similar. A std::initializer_list felt right because that's what we were using originally (though we used it implicitly to construct a vector and now we just use it straight up, without constructing a vector).

I think one drawback to std::array is that you have to specify the size at compile time if the type cannot be deduced:
```
std::initializer_list<EventTarget*>({ this })
// vs
std::array<EventTarget*, 1>({ this })
// vs
std::to_array<EventTarget*>({ this }) // Not sure if the playstation/windows port supports this yet. It didn't last I checked.
```